### PR TITLE
[WIP] r.out.maxent_swd: port to GRASS7 / Python3

### PR DESCRIPTION
**Pull request by ninsbl**

### CODE REVIEW

File: grass7/raster/r.out.maxent_swd/r.out.maxent_swd.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+A background SWD file is allways produced while specie output can be omitted.<br>
+<br>
+Multiple species can be processed, but each has to be in an
+individual raster map. Map names of the maps containing the environmental
+parameters can be replaced by short names, which should be used
+in MaxEnt 3.3.3.e.<br>
+<br>
+Results from MaxEnt can either be imported using r.in.xyz or
+calculated from MaxEnt lambdas file using the script
+r.maxent.lambdas.
```
Comment (neteler) [inline grass7/raster/r.out.maxent_swd/r.out.maxent_swd.html:24]:
```suggestion
represents one environmental parameter. In contrast to <b>r.stats</b>
only integer values are accepted to represent NO DATA.<br>
<br>
A background SWD file is always produced while specie output can be omitted.<br>
<br>
Multiple species can be processed, but each has to be in an
individual raster map. Map names of the maps containing the environmental
parameters can be replaced by short names, which should be used
in MaxEnt 3.3.3.e.<br>
<br>
Results from MaxEnt can either be imported using <b>r.in.xyz</b> or
calculated from MaxEnt lambdas file using the script
<b>r.maxent.lambdas</b>.
```

(using bold for command names)

File: grass7/raster/r.out.maxent_swd/r.out.maxent_swd.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+<h2>NOTES</h2>
+
+<h2>SEE ALSO</h2>
+
+<em>
+<a href="r.stats.html">r.stats</a>
+</em><br>
+<em>
+<a href="r.what.html">r.what</a>
+</em>
```
Comment (neteler) [inline grass7/raster/r.out.maxent_swd/r.out.maxent_swd.html:35]:
```suggestion
<em>
<a href="r.stats.html">r.stats</a>,
<a href="r.what.html">r.what</a>
</em>
```

File: grass7/raster/r.out.maxent_swd/r.out.maxent_swd.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+and <a href="https://github.com/mrmaxent/Maxent">https://github.com/mrmaxent/Maxent</a><br>
+MaxEnt 3.3.3e <a href="http://biodiversityinformatics.amnh.org/open_source/maxent/">http://biodiversityinformatics.amnh.org/open_source/maxent/</a><br>
+Wilson, Peter D. 2009: Guidlelines for computing MaxEnt model output values from a lambdas file. (Avaliable at <a href="http://groups.google.com/group/MaxEnt">
+http://groups.google.com/group/MaxEnt</a>)
+Steven J. Phillips, Miroslav Dudik, Robert E. Schapire. 2004: A maximum entropy approach to species distribution modeling. In Proceedings
+of the Twenty-First International Conference on Machine Learning, pages 655-662, 2004.<br>
+Steven J. Phillips, Robert P. Anderson, Robert E. Schapire. 2006: Maximum entropy modeling of species geographic distributions.
+Ecological Modelling, 190:231-259, 2006.<br>
+Jane Elith, Steven J. Phillips, Trevor Hastie, Miroslav Dudik, Yung En Chee, Colin J. Yates. 2011: A statistical explanation of MaxEnt
+for ecologists. Diversity and Distributions, 17:43-57, 2011.
```
Comment (neteler) [inline grass7/raster/r.out.maxent_swd/r.out.maxent_swd.html:48]:
```suggestion
<ul>
<li>Steven J. Phillips, Miroslav Dudík, Robert E. Schapire. 2020: Maxent software for modeling species niches and distributions (Version 3.4.1).
Available at: <a href="http://biodiversityinformatics.amnh.org/open_source/maxent">http://biodiversityinformatics.amnh.org/open_source/maxent</a>
and <a href="https://github.com/mrmaxent/Maxent">https://github.com/mrmaxent/Maxent</a></li>
<li>MaxEnt 3.3.3e <a href="http://biodiversityinformatics.amnh.org/open_source/maxent/">http://biodiversityinformatics.amnh.org/open_source/maxent/</a></li>
<li>Wilson, Peter D. 2009: Guidelines for computing MaxEnt model output values from a lambdas file. (Available at <a href="http://groups.google.com/group/MaxEnt">
http://groups.google.com/group/MaxEnt</a>)</li>
<li>Steven J. Phillips, Miroslav Dudik, Robert E. Schapire. 2004: A maximum entropy approach to species distribution modeling. In Proceedings
of the Twenty-First International Conference on Machine Learning, pages 655-662, 2004.</li>
<li>Steven J. Phillips, Robert P. Anderson, Robert E. Schapire. 2006: Maximum entropy modeling of species geographic distributions.
Ecological Modelling, 190:231-259, 2006.</li>
<li>Jane Elith, Steven J. Phillips, Trevor Hastie, Miroslav Dudik, Yung En Chee, Colin J. Yates. 2011: A statistical explanation of MaxEnt
for ecologists. Diversity and Distributions, 17:43-57, 2011.</li>
```

**Comment by ninsbl** [comment]
Replaced by #681

**Pull request closed without merging**